src/pyhf/schema/loader.py

```diff
@@ -54,9 +55,3 @@ def load_schema(schema_id: str):
             schema = json.load(json_schema)
             variables.SCHEMA_CACHE[schema['$id']] = schema
         return variables.SCHEMA_CACHE[schema['$id']]
-
-
-# pre-populate the cache to avoid network access
```

Comment:
NB: now works with local paths as expected (this is a change in `jsonschema` resolving). This was initially added back in #1917 by @lhenkelm , but it should not be needed anymore now.

Reply:
See #1980 as well.